wen-restart/src/wen_restart.rs

```diff
@@ -62,6 +70,9 @@ impl std::fmt::Display for WenRestartError {
             WenRestartError::BlockNotFound(slot) => {
                 write!(f, "Block not found: {}", slot)
             }
+            WenRestartError::BlockNotFrozenAfterReplay(slot, err) => {
```

Comment:
We did thiserror a while ago, but decided to switch to anyhow because we wanted backtrace:
https://github.com/solana-labs/solana/pull/33892#discussion_r1506762228

Could I do thiserror and enable backtrace?

Reply:
Oh neat, I didn't know about `anyhow` / backtracing functionality, and didn't really notice that is what is in use in this crate.

> Could I do thiserror and enable backtrace?

Good question and I'm not sure, would need to dig around or try a quick experiment